spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -170,6 +171,15 @@ def get_parser():
             Each label is a single voxel, which value corresponds to the spinal level (e.g.: 2 for spinal level 2). If you are using more than 2 labels, all spinal levels covering the region of interest should be provided (e.g., if you are interested in levels C2 to C7, then you should provide spinal level labels 2,3,4,5,6,7)."
         """)  # noqa: E501 (line too long)
     )
+    optional.add_argument(
+        '-lrootlet',
+        metavar=Metavar.file,
+        help=textwrap.dedent("""
+            Dorsal nerve rootlets segmentation from C2 to C8  Example: `anat_rootlets.nii.gz`
```

Comment:
maybe we can now drop "C2 to C8" from this line as the following line details the C2-C8 range
```suggestion
            Dorsal nerve rootlets segmentation. Example: `anat_rootlets.nii.gz`
```

Reply:
Agreed!